### PR TITLE
Integration improvements

### PR DESCRIPTION
**Pull request by ntunin**
It was difficult enough to integrate FirebaseBroad to the app that already use Firebase. To fix it there was removed Firebase import to FirebaseBroad

### CODE REVIEW

File: FirebaseBroad/FirebaseBroad.swift [hunk 1]
```diff
@@ -5,19 +5,30 @@
...
 import ButterBroad
-import Firebase
+import Foundation
 
 final public class FirebaseBroad: ButterBroad.Analytics {
 
-    public var activationHandler: (() -> Void)? = {
-        FirebaseApp.configure()
+    public lazy var activationHandler: (() -> Void)? = {
     }
```
Comment (Antowkos) [inline FirebaseBroad/FirebaseBroad.swift:13]: Why did you remove `FirebaseApp.configure()` call? It change consistency of broads behavior. Is it forced by Firebase behavior?

File: FirebaseBroadExample/Cartfile [hunk 2]
```diff
@@ -1,2 +1,2 @@
 github "rosberry/ButterBroad"
-github "rosberry/CrashlyticsBroad"
+github "rosberry/CrashlyticsBroad" "hotfix/isd-326"
```
Comment (Antowkos) [inline FirebaseBroadExample/Cartfile:2]: Not sure if it should get into master

File: Scripts/run [hunk 3]
```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```
Comment (Antowkos) [inline Scripts/run:1]: This file is pretty strange. Is it necessary? Same for `upload-symbols`

File: README.md [hunk 4]
```diff
@@ -14,32 +14,58 @@
...
+
 #### Creating a broad
 
 ```swift
 import ButterBroad
 import FirebaseBroad
 
 extension Butter {
-    static let firebase: FirebaseBroad = .init()
+static let firebase: FirebaseBroad = .init(Firebase.Analytics.self)
```
Comment (Antowkos) [inline README.md:68]: Wrong indentation and usage of `Firebase.Analytics.self` is strange. Can't we use it inside of the broad?

**Review by Antowkos — CHANGES_REQUESTED** [review]

**Comment by zhvrnkov** [comment]
Also can you please describe the issue with more details? Is this is a problem, where Broad (as separate target), configures its own Firebase?

File: Scripts/run (hunk 3 above)
Comment (ntunin) [reply Scripts/run:1]: No, but it simplifies the usage. Firebase Crashlytics framework builded with carthage should have this files in the package, but it don't have.

File: FirebaseBroadExample/Cartfile (hunk 2 above)
Comment (ntunin) [reply FirebaseBroadExample/Cartfile:2]: Before merge it is necessary

File: README.md (hunk 4 above)
Comment (ntunin) [reply README.md:68]: Yes, strange, but it is necessary to make Firebase Analytics instance initialized in the context that FirebaseBroad should not  have direct references to firebase

File: FirebaseBroad/FirebaseBroad.swift (hunk 1 above)
Comment (ntunin) [reply FirebaseBroad/FirebaseBroad.swift:13]: Current solution implies that FirebaseBroad has not direct references to firebase. More over, some kind of developers want to call `FirebaseApp.configure()` from their own code

**Comment by ntunin** [comment]
The problem appeared in Zazzi project. The call of `FirebaseApp.configure ()` from the developer code led to the crash even without this call in the `FirebaseBroad` instance. It was discovered, that some firebase dependency is refered to uninitialized default `FirebaseApp` instance while the app has initialized one. The problem was solved by removing of all explicit references to firebase in the `FirebaseBroad`
